internal/activities/create_sip.go

```diff
@@ -24,15 +22,8 @@ type SipCreationResult struct {
 }
 
 func (sc *SipCreationActivity) Execute(ctx context.Context, params *SipCreationParams) (*SipCreationResult, error) {
+	// TODO change this activity into a bag activity
 	res := &SipCreationResult{}
-	e, err := exec.Command("python3", "repackage_sip.py", params.SipPath).CombinedOutput() // #nosec G204
-	if err != nil {
-		return nil, err
-	}
-	res.Out = string(e)
-	if res.Out != params.SipPath+"_bag\n" {
-		return nil, errors.New("Failed to repackage sip correctly: " + res.Out)
-	}
 
 	res.NewSipPath = params.SipPath + "_bag"
```

Comment:
@Diogenesoftoronto, this path won't match without running the Python script. I'd suggest you remove the entire activity and add the to do comment to the workflow code.